src/plugins/betterDiscordThemeLoader/index.tsx

```diff
@@ -0,0 +1,44 @@
+import definePlugin from "@utils/types";
+import "./BetterDiscordThemesTab";
+import BetterDiscordThemesTab from "./BetterDiscordThemesTab";
+
+export default definePlugin({
+    name: "BetterDiscord Theme Loader",
```

Comment:
this name is misleading
```suggestion
    name: "BDThemeRepo",
```